src/helpers/index.ts

```diff
@@ -0,0 +1,82 @@
+import type { Pokemon } from '@/composable/getPokemon'
+import axios from 'axios'
+export const imageUrl = 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/'
+
+export const APIURL = 'https://pokeapi.co/api/v2/pokemon/'
+export const MAX_POKEMON = 1302
+export const sizeOptions = [
+  { name: '25', val: 25 },
+  { name: '50', val: 50 },
+  { name: '100', val: 100 },
+  { name: 'ALL', val: MAX_POKEMON }
+]
+
+export const getPageNumber = (): number => {
+  const pageNumber = localStorage.getItem('pageNumber')
+  if (pageNumber === null) {
+    return 1
+  } else {
+    return parseInt(pageNumber)
+  }
+}
+
+export const setPageNumber = (pageNumber: number) => {
+  localStorage.setItem('pageNumber', String(pageNumber))
+}
+
+export const getPagination = (): { name: string; val: number } => {
```

Comment:
`const pagination = localStorage.getItem('pagination');
  return pagination ? JSON.parse(pagination) : { name: '25', val: 25 };`

Reply:
It's usually a good idea t wrap JSON.parse() in a try/catch if you assume any chance it could fail